### DISCUSSION TITLE
Donors where IsDonorPhenotypeUnrepresented is true do not have value for MatchProbabilitiesPerLocus.OverallMatchCount

### DISCUSSION THREAD

**Discussion by mmelchers** [Anthony-Nolan/Atlas#910] [Q&A] [answered]
**Describe the bug**
As a search coordinator I expect that even "inexplicable" donors (where IsDonorPhenotypeUnrepresented= true) have a match class assigned. This is currently not the case for ATLAS

**To Reproduce**
Steps to reproduce the behaviour:
1. Start a search with many search results
2. Find a donor where  IsDonorPhenotypeUnrepresented= true
3. check value for MatchProbabilitiesPerLocus.OverallMatchCount

**Expected behaviour**
MatchProbabilitiesPerLocus.OverallMatchCount has a numerical value

**Actual behaviour**
MatchProbabilitiesPerLocus.OverallMatchCount = null

**Additional context**
Since we are using the match probabilities from ATLAS, we were told to also use the match probability based overallMatchCount

**Comment by zabeen** [comment, marked as answer]
Discussed this with @mmelchers 

Atlas cannot determine a predicted match count value where it was unable to calculate match probabilities due to either patient or donor being unrepresented by the HLA HF set.

The consumer service can instead fall back on the [potential match count, which is based on P groups](https://github.com/Anthony-Nolan/Atlas/blob/947174ba820f63dba64beb4d7092f118b799f0a6/Atlas.Client.Models/Search/Results/Matching/MatchingAlgorithmResult.cs#L30).
